specs/commandline/sign.md

```diff
@@ -31,6 +41,7 @@ Flags:
        --allow-referrers-api        [Experimental] use the Referrers API to store signatures in the registry, if not supported (returns 404), fallback to the Referrers tag schema
   -d,  --debug                      debug mode
   -e,  --expiry duration            optional expiry that provides a "best by use" time for the artifact. The duration is specified in minutes(m) and/or hours(h). For example: 12h, 30m, 3h20m
+       --file                       enable signing a file located in file system, if set, the reference argument is the file path (required if --signature is set)
```

Comment:
We should also support reading data from stdin or as an parameter ?

Reply:
@priteshbandi I think we could discuss this further, but it's kind of another topic than `sign/verify a file` in this PR (another flag, maybe `--stdin`, needs to be introduced in our future iterations).

Reply:
Tracking this discussion here: https://github.com/notaryproject/notation/pull/765#discussion_r1301119361